java-client/build.gradle.kts

```diff
@@ -122,13 +123,14 @@ val integrationTest = task<Test>("integrationTest") {
 
 dependencies {
 
-    val opensearchVersion = "1.2.4"
+    val opensearchVersion = "2.0.0-alpha1-SNAPSHOT"
     val jacksonVersion = "2.12.6"
     val jacksonDatabindVersion = "2.12.6.1"
 
     // Apache 2.0
     implementation("org.opensearch.client", "opensearch-rest-client", opensearchVersion)
     testImplementation("org.opensearch.test", "framework", opensearchVersion)
+    implementation("org.apache.logging.log4j", "log4j-core", "2.17.2")
```

Comment:
Is this introducing a hard-coded dependency on the Log4j framework?  Surely the client should only be dependent on the facade?

Reply:
#175